docs/4-csv-format.md

```diff
@@ -15,7 +15,7 @@ ActiveAdmin.register Post do
   csv do
     column :title
     column(:author) { |post| post.author.full_name }
-    column('bODY', humanize_name: false) # preserve case
+    column('body', humanize_name: false) # preserves case of column title
```

Comment:
@javierjulio, this was a fair use of 'bODY'. The "typo" + the comment shows that you can customize the title the way you need. The new comment looks good, I'm just talking about leaving 'bODY' as an exception of the new flow.

![image](https://github.com/user-attachments/assets/1a1c19f4-61dd-4879-b189-6ed1764d7324)

Reply:
@mgrunberg yes, I know but I was cheating because its all lowercase and by default will capitalize so I think that was sufficient. Let me look into another update to ignore this specific word in only this file.